Alignment/APEEstimation/test/cfgTemplate/apeEstimator_cfg.py

```diff
@@ -159,6 +162,7 @@
 #process.source.duplicateCheckMode = cms.untracked.string("checkAllFilesOpened")   # default value
 
 
+import CalibTracker.Configuration.Common.PoolDBESSource_cfi
```

Comment:
when you use this module you will get a deprecation warning
see more comments below